x-pack/packages/kbn-data-forge/src/run.ts

```diff
@@ -23,5 +23,6 @@ export async function run(config: Config, client: Client, logger: ToolingLog) {
   await indexSchedule(config, client, logger);
   const indicesCreated = [...indices];
   indices.clear();
+  await client.indices.refresh({ index: indicesCreated });
```

Comment:
This refresh index after generating data-forge data helped with decreasing the time of running tests and making sure data is available when the rule is executed, which in one instance, the time of the tests decreased from `30.0s` to `15.7s` 🎉 

|Before|After|
|---|---|
|![image](https://github.com/elastic/kibana/assets/12370520/173ab47c-3d23-4e88-bb80-c60d59a3b4bd)|![image](https://github.com/elastic/kibana/assets/12370520/e6bedd5c-c920-4bc5-ab76-91f3e420aa1e)|
|![image](https://github.com/elastic/kibana/assets/12370520/31adbaa4-ada0-4d80-be29-05be8f72818a)|![image](https://github.com/elastic/kibana/assets/12370520/75b398d1-53d9-44b0-bbdf-b8826d25de69)|